libres/lib/enkf/enkf_obs.cpp

```diff
@@ -318,10 +318,7 @@ static void enkf_obs_get_obs_and_measure_summary(
             break;
 
         if (local_obsdata_node_tstep_active(obs_node, step) &&
-            obs_vector_iget_active(obs_vector, step) &&
-            active_list_iget(
-                active_list,
-                0 /* Index into the scalar summary observation */)) {
```

Comment:
This PR looks like a good change 👍 But is there no way for the `active_list` to be partially active at this point?

Reply:
> But is there no way for the active_list to be partially active at this point?

No - because this is in a special function for assembling *summary* observations, and they are scalar, i.e. `PARTLY_ACTIVE == ALL_ACTIVE`

Reply:
Right, and because we are only using `obs_vector_iget_active` they cant be inactive?

Reply:
No not exactly - it is more that since the summary observation only has one single value, if it is at all present in the observation set it has to be active: When it is constructed it is in the default `ALL_ACTIVE` mode, and the only index you can explicitly activate is index 0, then you get a `PARTLY_ACTIVE` with the one-and-only element active - i.e. identical to `ALL_ACTIVE` (NB: we can not deactivate an index).

Reply:
Thanks for the explanation 👍